### DISCUSSION TITLE
Journalctl From System-Status Not Working

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#937] [Fixed Issues]
@rern 

Clicking on the Status cogs does not update the systemctl display.

Would it be possible for this to show just the current and previous boot info?

rAudio 1 • 20220526
5.15.44-1 • aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 952MiB
4 x Cortex-A53 @ 1.2 GHz

**Comment by R101G** [comment]
Status still not showing current journalctl info after 20220618 update. 

Is it just me?

**Comment by ceejayemm** [comment]
Clicking on the Cog to the right of the Status item gives me a great long list of information (RPi3A and latest rAudio-1 update 22020618) but doesn't change much after startup completes. Clicking the circular arrows to the right again of the cog turns on a (10 sec) refresh of Status - circular arrows flash while activated. Question Mark to far right gives details. With circular arrows turned on the Status does get updated for me, see screen shots attached, you will need to look closely to see the difference but it does change (CPU Load, Time and Uptime).

Chris

![status1](https://user-images.githubusercontent.com/25418025/174994263-4586e233-9ceb-43cd-9dc1-c7306ed24117.JPG)
![status2](https://user-images.githubusercontent.com/25418025/174994287-a2508bf7-ddb0-45f1-b3a3-31b04f71f96f.JPG)
![status3](https://user-images.githubusercontent.com/25418025/174994320-5169ca20-8c63-415d-8e79-7dc7962f925e.JPG)

**Comment by R101G** [comment]
@ceejayemm 
Do you get the current boot info displayed?

Clicking on the cogs should run the journalctl -b command, In my case it shows the command but just displays the same very old log entry. Running the command from the terminal works as it should, so it looks like a UI problem.

Whether the feature is important is arguable but, as it is there, it ought to work!

**Comment by ceejayemm** [comment]
@R101G 

The Cog does a 'journalctl -b' which in my case reports data from May 23 this year (when I flashed the SD card) through to June 13th (when I last rebooted by RPi). Running journalctl -b from a terminal session reports data thru to right now. So I guess rAudio-1 is taking a snapshot of the command results at boot time and displaying that.  

I am not sure that I would want rAudio-1 trying to keep the output up to date as: 

(1) I hardly ever use it and would be happy to get this information via a terminal session if I do
(2) it will take resources I would rather use for music output :-)

Chris

**Comment by rern** [comment]
The bug has been found. To be fixed by next update.

**Comment by R101G** [comment]
Thanks @ceejayemm. It looks as if yours is showing the last boot info as it should. And thanks to @rern for the forthcoming fix.

**Comment by R101G** [comment]
Thanks for looking at this @rern. It now displays the current boot plus a previous boot showing a random date. I thought this might be due to the SD card having been moved around different Pi versions, so I tried a new image. The initial boot showed the information OK. The second boot showed the current boot OK plus the previous boot with May 23rd as the date. The third boot showed the current boot OK plus the previous boot with a date of June 3rd.
